bublik/interfaces/api_v2/config.py

```diff
@@ -3,10 +3,12 @@
 
```

Comment:
Don't use past tense in the summary.

Also summary is full is technical details which are very-very specific and reader may not know it.
Just say why the change is needed in the summary. Which problems is solved? Ideally without method names etc.
Technical details are allowed in the description, but not summary.

Reply:
Improved summary

Reply:
Summary is too long. I'd just drop "with custom `filterset_class"

Reply:
Done